python/lsst/ts/lasertracker/config_schema.py

```diff
@@ -52,6 +52,20 @@
         read_timeout:
           description: Timeout for reading T2SA replies to commands (seconds).
           type: number
+        m2_zeropoint:
+          description: zeropoint array for M2 [x, y, z, rx, ry].
+          type: array
```

Comment:
Consider making a `zeropoint` session and having m2, cam as submissions with xyzuv as subsubsessions.

Something like:

```
zero_points:
description: Zero points for components positions:
type: object
properties:
  m2:
    description: Zero point for M2.
    type: object
    properties:
      x:
        description: Zero point in x in mm. 
        type: number
      y:
        description: Zero point in y in mm. 
        type: number
      z:
        description: Zero point in z in mm. 
        type: number
      u:
        description: Zero point for rotations around x axis in degrees. 
        type: number
      v:
        description: Zero point for rotations around y axis in degrees. 
        type: number
  camera:
    description: Zero point for M2.
    type: object
    properties:
      x:
        description: Zero point in x in mm. 
        type: number
      y:
        description: Zero point in y in mm. 
        type: number
      z:
        description: Zero point in z in mm. 
        type: number
      u:
        description: Zero point for rotations around x axis in degrees. 
        type: number
      v:
        description: Zero point for rotations around y axis in degrees. 
        type: number
```